ffv1.md

```diff
@@ -279,24 +279,32 @@ For each plane of each slice, a Quantization Table Set is selected from an index
 
 Background: in first implementations of FFV1 bitstream, the index for Cb and Cr planes was stored even if it is not used (chroma_planes set to 0), this index is kept for version <= 3 in order to keep compatibility with FFV1 bitstreams in the wild.
 
-## Color space
+## Components
 
-FFV1 supports two color spaces: YCbCr and JPEG2000-RCT. Both color spaces allow an optional Alpha plane that can be used to code transparency data.
+FFV1 supports 1 Luma component, optionaly 2 Chroma components and 1 Alpha component that can be used to code transparency data.
 
-### YCbCr
+## Pixel transformations
 
-In YCbCr color space, the Cb and Cr planes are optional, but if used then MUST be used together. Omitting the Cb and Cr planes codes the frames in grayscale without color data. An FFV1 `Frame` using YCbCr MUST use one of the following arrangements:
+Transformations permit to modify the pixel data in a reversible way in order to improve the compression ratio, before being compressed. An example of usage is to compress RGB content which could have each component independently compressed but with a less optimal compression ratio.
```

Comment:
re "in order to improve the compression ratio" is this why? Aren't there transformations to allow FFV1 to support non-YCbCr colorspaces (RGB and maybe someday Bayer formats). Suggesting maybe: "Transformations permit to modify the pixel data in a reversible way in order to store alternate color spaces"

Reply:
RGB could be compressed without transformation, just saying that order is RGB instead of YCbCr, transformation is here for efficient compression.

I could remove the explanation if needed.